docs/clusterdefinition.md

```diff
@@ -161,8 +161,9 @@ More usefully, let's add some custom configuration to the above addons:
               }
             ],
             "config": {
-              "maxNodes": "5",
-              "minNodes": "1"
+              "max-nodes": "5",
+              "min-nodes": "1",
+              "scan-interval": "10s"
```

Comment:
There's another instance of `maxNodes` in cluster-autoscaler config at https://github.com/Azure/aks-engine/blob/master/pkg/engine/testdata/addons/kubernetes.json#L39-L54 that maybe should be fixed up similarly.